sdk/identity/identity/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - Added support to specify `subscription` ID or name on `AzureCliCredential` error message. [#34801](https://github.com/Azure/azure-sdk-for-js/pull/34801)
+- Added support in `AzurePowerShellCredential` for the Az.Accounts v5.0.0+ and Az v14.0.0+ breaking change where `Get-AzAccessToken` returns `SecureString PSSecureAccessToken.Token` instead of plaintext `PSAccessToken.Token`. [#34902](https://github.com/Azure/azure-sdk-for-js/pull/34902)
```

Comment:
I'm convinced that `AzurePowerShellCredential` already supported Az.Accounts v5.0.0+ and Az v14.0.0+ in the latest version of this SDK since we would always pass `-AsSecureString` if the Az.Accounts version was >= 2.17.0, thus `Get-AzAccessToken` would always return a secure string if the version was >= 2.17.0.

v5.0.0+ returning a secure string by default would just be the same behavior we were already expecting. 

The change here is more for supporting older versions of PowerShell. How about updating the changelog to something like:
```
- Fixed an issue with `AzurePowerShellCredential` not working correctly for users using older versions of PowerShell (e.g., Windows PowerShell 5.1) where `-AsPlainText` is not supported in the `ConvertFrom-SecureString` cmdlet.
```

Reply:
Updated!